src/builtins/core/year_month.rs

```diff
@@ -103,11 +102,19 @@ impl PlainYearMonth {
         self.iso.year
     }
 
-    /// Returns the padded ISO year string
+    /// 3.5.11 PadISOYear ( y )
+    ///
+    /// Returns a String representation of y suitable for inclusion in an ISO 8601 string.
     #[inline]
     #[must_use]
     pub fn padded_iso_year_string(&self) -> String {
-        pad_iso_year(self.iso.year)
```

Comment:
I think this can actually be removed. We're not using this in `Boa` and `to_ixdtf_string` supersedes it. For some reason, I didn't notice this was still part of the public API.